client/src/components/styles/ReviewEntry.styles.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  box-sizing: border-box;
+  border-bottom: .5px solid #ccc;
+  max-width: 650px;
+`;
+
+export const ReviewUser = styled.div`
+  padding-top:24px;
+  font-weight: 600;
+  color: #484848;
+  font-family: Circular,BlinkMacSystemFont,Helvetica Neue,sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: inherit;
+  -webkit-font-smoothing: antialiased;
+`;
+
+export const ReviewDate = styled.div`
+  color: #484848;
+  font-family: Circular,BlinkMacSystemFont,Helvetica Neue,sans-serif;
+  font-size: 14px;
+  line-height: 18px;
+  box-sizing: inherit;
+  -webkit-font-smoothing: antialiased;
+  padding-top: 2px;
+`;
+
+export const ReviewText = styled.div`
+  color: #484848;
+  font-family: Circular,BlinkMacSystemFont,Helvetica Neue,sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: inherit;
+  -webkit-font-smoothing: antialiased;
+  padding-top: 22px;
+  padding-bottom: 24px;
+`;
+
```

Comment:
Since most of these have the same properties, I recommend using styled component's extend functionality, or styling based on props. You can have a Review styled.div, and extend it based on the name. But, you've already created yours, so this would just be a refactor.

Reply:
Love that you pointed this out - I hadn't stumbled upon extend but I just looked it up and it makes a ton of sense!  I like it.  I'll look into a re-factor.